app/locale/af/LC_MESSAGES/django.po

```diff
@@ -1371,9 +1371,6 @@ msgstr "Status van persoon"
 msgid "Status of this person:"
 msgstr "Status van hierdie persoon:"
 
-msgid "Street name"
-msgstr "Straatnaam"
-
 msgid "Street name only, no number"
```

Comment:
just checking, should we also remove "Street name only, no number" ?

Reply:
Yeah, that string was a hint for the street name field, so we don't need it anymore.